### PR TITLE
Editorial suggestions from Ben Kaduk's review

### PR DESCRIPTION
**Pull request by kaduk**
I suspect that there is *somewhere* that I was confused about things and introduced a bad change, so please check before merging.

### CODE REVIEW

File: draft-ietf-grow-bmp-local-rib.xml [hunk 1]
```diff
@@ -290,14 +290,16 @@
...
-                    peers have link-state address family enabled.
+                    peers have link-state address family enabled are all
+                    situations with no preexisting BGP peer where Loc-RIB
+                    access via BMP would be useful.
           </li>
         </ul>
         <t>
-                    Complexities introduced when correlating a received Adj-RIB-In as
-                    a router Loc-RIB:
+                    Many complexities are introduced when using a received Adj-RIB-In
```
Comment (TimEvens) [inline draft-ietf-grow-bmp-local-rib.xml:300]: nice change.

File: draft-ietf-grow-bmp-local-rib.xml [hunk 2]
```diff
@@ -290,14 +290,16 @@
                     Requires additional BGP peering to collect the received updates
                     when peering may have not even been required in the first
                     place. For example, virtual routing and forarding (VRF) tables 
-                    with no peers, redistributed BGP-LS with no peers,
+                    with no peers, redistributed BGP-LS with no peers, and
                     segment routing egress peer engineering where no
-                    peers have link-state address family enabled.
+                    peers have link-state address family enabled are all
+                    situations with no preexisting BGP peer where Loc-RIB
```
Comment (TimEvens) [inline draft-ietf-grow-bmp-local-rib.xml]: This looks like a good change, but "where Loc-RIB access via BMP would be useful." should be dropped.  The example is where we do not have preexisting peers and having to add one just to collect data is suboptimal and wasteful (IMHO).

**Review by TimEvens — CHANGES_REQUESTED** [review]
This looks great, with the one exception on the example.  I can make that update and merge. Please let me know if you agree.

**Comment by kaduk** [comment]
> This looks great, with the one exception on the example. I can make that update and merge. Please let me know if you agree.

Sure, please go ahead with your proposed plan.

**Pull request merged**